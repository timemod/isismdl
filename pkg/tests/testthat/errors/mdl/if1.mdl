ident x1 = if (x = 2) then 3;
ident x2 = if (x = 2) then 3 endif;
ident x3 = if (x = 2) else 2;
ident x4 = if (x = 2) endif;
