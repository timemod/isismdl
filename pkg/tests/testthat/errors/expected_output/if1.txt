syntax error, unexpected ';', expecting else
Line: 1 of file mdl/if1.mdl
ident x1 = if (x = 2) then 3;
                            ^

syntax error, unexpected endif, expecting else
Line: 2 of file mdl/if1.mdl
ident x2 = if (x = 2) then 3 endif;
                                  ^

syntax error, unexpected else
Line: 3 of file mdl/if1.mdl
ident x3 = if (x = 2) else 2;
                          ^

syntax error, unexpected endif
Line: 4 of file mdl/if1.mdl
ident x4 = if (x = 2) endif;
                           ^

